src/sessions/PageManager.ts

```diff
@@ -165,6 +167,7 @@ export class PageManager {
             pageId,
             parentPageId: currentPage.pageId,
             interaction: currentPage.interaction + 1,
+            timeSpentOnParentPage: timeOnParentPage,
```

Comment:
question: What happens when the page is resumed?

We may want to change the behavior so that page view events are not recorded for resumed pages. RUM users have been confused by seeing users navigate from "PageA" to "PageA" in user journeys.

Reply:
Handled this case in revision. Page view events are no longer recorded for resumed pages if page ID is same. Start time from the Session cookie is used to compute the time on page if resumed page is different than when it was when session stopped.